doc/source/user_guide/visualization.rst

```diff
@@ -117,10 +116,10 @@ For example, a bar plot can be created the following way:
 
 .. ipython:: python
 
-   plt.figure();
```

Comment:
I think these semicolons may have been added intentionally to suppress certain output, I'd leave them for the purposes of this

Reply:
Okay, I returned all the semicolons to the `plt.figure()` lines that had them.